docs/_components.scss

```diff
@@ -803,4 +803,84 @@
     100% {
         @include transform(rotate(360deg));
     }
+}
+
+// Page Footer
+//
+// markup:
+// <footer>
+//      <ul class="footer-links columns is-mobile">
+//          <li class="column"><a href="#">About</a></li>
+//          <li class="column"><a href="#">FAQ</a></li>
+//          <li class="column"><a href="#">Contact</a></li>
+//          <li class="column"><a href="#">Privacy</a></li>
+//      </ul>
+//      <h4 class='footer-social__heading'>Follow us</h4>
+//      <ul class="footer-social columns is-mobile">
+//          <div class="column">
+//              <li><a href="#"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 400 400" class="icon_twitter"><path class="icon_twitter--icon" d="M199.85.44c-110.46 0-200 89.54-200 200s89.54 200 200 200 200-89.54 200-200-89.54-200-200-200zm45.2 104.5c13.87 0 26.4 5.87 35.2 15.24 10.98-2.16 21.3-6.18 30.6-11.7-3.6 11.25-11.23 20.7-21.2 26.67 9.76-1.1 19.06-3.7 27.7-7.5-6.46 9.68-14.64 18.17-24.05 24.98.1 2.07.14 4.15.14 6.23 0 63.7-48.5 137.2-137.2 137.2-27.3 0-52.6-8-73.9-21.66 3.7.44 7.6.66 11.5.66 22.6 0 43.3-7.7 59.8-20.65-21.1-.4-38.9-14.3-45.06-33.5 2.94.6 5.96.9 9.06.9 4.4 0 8.67-.6 12.7-1.7-22.04-4.4-38.7-23.9-38.7-47.22 0-.22 0-.42.05-.62 6.5 3.64 13.9 5.8 21.8 6.05-12.94-8.6-21.45-23.4-21.45-40.1 0-8.83 2.36-17.1 6.5-24.23 23.77 29.18 59.3 48.37 99.37 50.38-.85-3.5-1.28-7.2-1.28-10.97 0-26.64 21.6-48.2 48.2-48.2z" /></svg></a></li>
```

Comment:
I think font awesome has twitter, facebook and linkedin icons you can use rather than embedding svgs